src/users/user.controller.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { Test, TestingModule } from '@nestjs/testing'
+import { UserController } from './user.controller'
+import { UserService } from './user.service'
+
+describe('UserController', () => {
+  let app: TestingModule
+
+  beforeAll(async () => {
+    app = await Test.createTestingModule({
+      controllers: [UserController],
+      providers: [UserService],
+    }).compile()
+  })
+
+  describe('root', () => {
```

Comment:
⚠️ 
oops, it should be `user` instead of `root`

Reply:
😅